opentelemetry-otlp/examples/basic-otlp/src/main.rs

```diff
@@ -131,7 +132,7 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
 
             span.add_event("Sub span event", vec![]);
 
-            histogram.record(1.3, &[]);
+            histogram.record(1.3, AttributeSet::default());
```

Comment:
not in this PR, but just realized that this is somewhat awkward having to pass empty/default when no dimensions!

Reply:
This is kind of an artifact of the original state of the PR being backward incompatible.  In the current state of the code `&[]` now works again.